src/getAppSyncConfig.js

```diff
@@ -108,22 +107,16 @@ export default function getAppSyncConfig(context, appSyncConfig) {
 
         return {
           ...dataSource,
-          invoke: (payload) =>
-            invoke({
-              packageFolder: path.join(
-                context.serverless.config.servicePath,
-                context.options.location,
-              ),
-              handler: func.handler,
-              event: JSON.stringify(payload),
-              environment: {
-                ...(context.options.lambda.loadLocalEnv === true
-                  ? process.env
-                  : {}),
-                ...context.serverless.service.provider.environment,
-                ...func.environment,
-              },
-            }),
+          invoke: async (payload) => {
+            const result = await axios.request({
+              url: `http://localhost:${context.options.lambdaPort}/2015-03-31/functions/${func.name}/invocations`,
+              method: 'POST',
+              data: payload,
+              validateStatus: false,
+              headers: payload.request.headers,
```

Comment:
@seanvm I had an issue here where `request` was undefined. 

```suggestion
              headers: payload.request?.headers,
```